test/quantization/core/test_quantized_op.py

```diff
@@ -2269,7 +2271,7 @@ def test_empty_batch(self):
 
         # interpolate
         op = torch.nn.quantized.functional.interpolate
-        for mode in ["nearest", "bilinear"]:
+        for mode in ["nearest", "bilinear", "nearest-exact"]:
```

Comment:
I think this test change will have to be left out for now until nearest-exact mode is supported through F.interpolate